effects/effect/effect-module.nix

```diff
@@ -185,6 +185,52 @@ in
       default = 0;
     };
 
+    binsh = mkOption {
+      type = types.nullOr types.str;
+      description = ''
+        The target for the `/bin/sh` symlink.
+
+        If set to `null`, `/bin/sh` will not be created.
+      '';
+      defaultText = lib.literalMD ''
+        `lib.getExe pkgs.bash`, where `pkgs` is the `pkgs` argument that was [passed to the library](https://docs.hercules-ci.com/hercules-ci-effects/guide/import-or-pin).
+      '';
+      default = lib.getExe pkgs.bash;
+    };
+
+    usrbinenv = mkOption {
```

Comment:
I don't think this is needed by default. Shebangs in nix are usually patched and we have good automation for it. Patching your glibc on the other hand is not easy.

Reply:
It does make it a bit easier to run developer/maintainer scripts that aren't packaged, like nix shell shebang scripts.
Also for normal shell scripts, I'd rather have them write `#!/usr/bin/env bash` than give them an incentive to use `/bin/sh` for "simplicity".